config/routes.rb

```diff
@@ -120,6 +120,8 @@
       end
     end
     post 'solr/update' => 'solr#update'
+    resource :translations, only: [:edit, :update] do
+    end
```

Comment:
I think this block may be redundant.

Reply:
yep totally that is cruft

Reply:
Removed in 2059a7bd626815c13f5af3e8faf576b43d4777a2